values/time.go

```diff
@@ -212,6 +212,14 @@ func (d Duration) Nanoseconds() int64 { return d.nsecs }
 // It will ensure that the output duration is the smallest positive
 // duration that is the equivalent of the current duration.
 func (d Duration) Normalize(interval Duration) Duration {
+	if d.Months() != 0 && d.Nanoseconds() == 0 &&
+		interval.Months() != 0 && interval.Nanoseconds() == 0 {
+		return Duration{
+			months:   d.Months() % interval.Months(),
```

Comment:
Below the nanosecond logic checks for negative offsets. Do we need to do the same here for months?

Reply:
I added a check for negative offsets.